app/src/main/java/com/esri/arcgismaps/kotlin/sampleviewer/model/room/SampleDao.kt

```diff
@@ -17,18 +16,14 @@ interface SampleDao {
     @Insert(onConflict = OnConflictStrategy.REPLACE)
     suspend fun insertAll(samples: List<SampleEntity>)
 
-    /**
-     * Flow is used here to observe any live changes to the samplesDB
-     */
-    @Query("SELECT rowid, name, codeFile, readMe, relevantAPIs FROM samplesDB")
-    fun fetchItems(): Flow<List<SampleEntity>>
-
+    // In support of ranked search.
     @Query(
         """SELECT name, codeFile, readMe, relevantAPIs, MATCHINFO(samplesDB, 'pcnalx') FROM samplesDB 
         WHERE samplesDB MATCH :query"""
     )
     fun matchInfoSearch(query: String): Cursor
 
+    // In support of search suggestions.
```

Comment:
Can you add the relevant comment for `getFilteredRelevantAPIs` query function as well :)